torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -657,23 +657,18 @@ def get_attr(
         ],
         fx_graph_module: torch.fx.GraphModule,
     ):
-        current_attr = fx_graph_module
-        sub_attr_names = node.target.split(".")  # type: ignore[union-attr]
-        # If node.targe is "conv.weight", the following loop first
-        # assigns fx_module_with_metadata.conv to current_attr, and then
-        # fx_module_with_metadata.conv.weight to current_attr.
-        while sub_attr_names:
-            sub_attr_name = sub_attr_names.pop(0)
-            if not hasattr(current_attr, sub_attr_name):
-                raise AttributeError(
-                    f"Attribute {sub_attr_name} is not found in {current_attr}."
-                )
-            current_attr = getattr(current_attr, sub_attr_name)
+        if not isinstance(node.target, str):
+            raise TypeError(
+                f"node.target must be a str, not {type(node.target)}. Found for node {node}."
+            )
+        assert isinstance(node.target, str), f"node.target {node.target} is not a str."
+        attr_tensor = getattr(fx_graph_module, node.target)
+        assert isinstance(attr_tensor, torch.Tensor), f"{attr_tensor} is not a tensor."
 
-        input_ = onnxscript_graph.add_initializer(node.name, current_attr)
+        input_ = onnxscript_graph.add_initializer(
+            node.target.replace("/", "."), attr_tensor
```

Comment:
Depending on how much people hate this `replace` pair. I can leave this as is or make a much more complicated PR storing the name mapping into a custom meta class inside node.meta, and try retrieve the name from there.

Reply:
lgtm